lib/dsl/lexer/lexer.js

```diff
@@ -16,44 +16,22 @@
  * limitations under the License.
  */
 
-const _ = require('lodash');
-const { Lexer, createToken } = require('chevrotain');
+const { Lexer } = require('chevrotain');
 const {
   NAME,
-  KEYWORD,
-  namePattern,
   sharedCategoryTokens: { UNARY_OPTION, BINARY_OPTION }
 } = require('./shared_tokens');
 const ValidationTokens = require('./validation_tokens');
 const ApplicationTokens = require('./application_tokens');
 const DeploymentTokens = require('./deployment_tokens');
 const RelationshipTypeTokens = require('./relationship_type_tokens');
 const OptionTokens = require('./option_tokens');
+const createTokenFromConfigCreator = require('./token_creator').createTokenFromConfig;
```

Comment:
I'd rather have `TokenCreator = ...`
and then below `TokenCreator.createTokenFromConfig`.

There's a reason I didn't do this refactoring when I could: I didn't want any reference to the new one just yet to avoid naming issues (even if they are small).

Reply:
I lost quite a time trying to understand createTokenFromConfigCreator.
Duplication makes worse.

Reply:
Yeah, it's really chevrotain-related... Alright then! thanks